spec/gelauto_spec.rb

```diff
@@ -23,6 +25,30 @@
     end
   end
 
+  context 'with Sorbet generic types' do
+    it 'uses T.nilable with T.any correctly' do
+      Gelauto.discover do
+        GelautoSpecs::Utility.safe_to_string(nil)
+        GelautoSpecs::Utility.safe_to_string(1.0)
+        GelautoSpecs::Utility.safe_to_string(2)
+      end
+
+      safe_to_string = get_indexed_method(GelautoSpecs::Utility, :safe_to_string)
+      expect(safe_to_string.to_sig).to include('input: T.nilable(T.any(Float, Integer))')
```

Comment:
Yeah, I think this is the easiest way to test `T.nilable`. Can we use `eq` instead of `include` here tho?

Reply:
Sure